src/lib/containers/entity_finder/tree/VirtualizedTree.tsx

```diff
@@ -267,6 +276,40 @@ export function Node(
     return <></>
   }
 
+  let tooltipContent: React.ReactNode = ''
+  if (isEntityHeaderNode(node)) {
+    tooltipContent = (
+      <div style={{ textAlign: 'center' }}>
+        <Typography display="inline" component="span" variant="smallText1">
+          {node.name}
+          <br />
+          <b>ID: </b>
+          {node.id}
+          {'modifiedBy' in node && (
```

Comment:
Note that we may not have `modifiedBy` or `modifiedOn`. That will happen if the node was retrieved using the entity path, which does not return those fields. Currently, the node that appears this way is the project in the "Current Project" configuration. The tooltip will just be missing the `modified[On|By]` fields.